OpenKh.Tools.ModsManager/Services/ConfigurationService.cs

```diff
@@ -1,511 +1,518 @@
+            set
+            {
+                _config.steamAPITrick1525 = value;
+                _config.Save(ConfigPath);
+            }
+        }
+        public static bool SteamAPITrick28
+        {
+            get => _config.steamAPITrick28;
+            set
+            {
+                _config.steamAPITrick28 = value;
+                _config.Save(ConfigPath);
+            }
+        }
+        public static bool Extractkh1
+        {
+            get => _config.GamesToExtract.Contains("kh1");
+            set
+            {
+                if (value)
+                {
+                    _config.GamesToExtract.Add("kh1");
+                }
+                else
+                {
+                    _config.GamesToExtract.Remove("kh1");
+                }
+                _config.Save(ConfigPath);
+            }
+        }
+        public static bool Extractkh2
+        {
+            get => _config.GamesToExtract.Contains("kh2");
+            set
+            {
+                if (value)
+                {
+                    _config.GamesToExtract.Add("kh2");
+                }
+                else
+                {
+                    _config.GamesToExtract.Remove("kh2");
+                }
+                _config.Save(ConfigPath);
+            }
+        }
+        public static bool Extractbbs
+        {
+            get => _config.GamesToExtract.Contains("bbs");
+            set
+            {
+                if (value)
+                {
+                    _config.GamesToExtract.Add("bbs");
+                }
+                else
+                {
+                    _config.GamesToExtract.Remove("bbs");
+                }
+                _config.Save(ConfigPath);
+            }
+        }
+        public static bool Extractrecom
+        {
+            get => _config.GamesToExtract.Contains("Recom");
+            set
+            {
+                if (value)
+                {
+                    _config.GamesToExtract.Add("Recom");
+                }
+                else
+                {
+                    _config.GamesToExtract.Remove("Recom");
+                }
+                _config.Save(ConfigPath);
+            }
+        }
+        public static bool Extractkh3d
+        {
+            get => _config.GamesToExtract.Contains("kh3d");
+            set
+            {
+                if (value)
+                {
+                    _config.GamesToExtract.Add("kh3d");
+                }
+                else
+                {
+                    _config.GamesToExtract.Remove("kh3d");
+                }
+                _config.Save(ConfigPath);
+            }
+        }
+        public static bool SkipRemastered
+        {
+            get => _config.SkipRemastered;
+            set
+            {
+                _config.SkipRemastered = value;
+                _config.Save(ConfigPath);
+            }
+        }
+        public static string LaunchGame
```

Comment:
Is this file needed to be edited anymore since the downloadable mod view is now its own window. My thought is you can have your own dropdown in the new window to filter which game to show (or maybe show all if the user wanted). and even if the launch game is chosen for KH2 you can install a mod for say KH1 and it will install into the correct folder.

Reply:
I may be wrong and it might be easier to still attach to LaunchGame if thats the case then so be it. idk what better attaching to LaunchGame or allowing ModsService.InstallMod to use a different path than the currently selected LaunchGame